contracts/core/TribeRoles.sol

```diff
@@ -59,6 +59,13 @@ library TribeRoles {
     /// @notice manages the constituents of Optimistic Timelocks, including Proposers and Executors
     bytes32 internal constant OPTIMISTIC_ADMIN = keccak256("OPTIMISTIC_ADMIN");
 
+    /// @notice manages the granting and revocation of PCV Guard roles
+    bytes32 internal constant PCV_GUARD_ADMIN =
+        keccak256("PCV_GUARD_ADMIN_ROLE");
+
+    ///@notice is able to withdraw whistelisted PCV deposits to a safe address
+    bytes32 internal constant PCV_GUARD = keccak256("PCV_GUARD_ROLE");
```

Comment:
this should be moved to the major roles section as it can unilaterally move funds